conjure_python_client/_http/requests_client.py

```diff
@@ -196,18 +201,43 @@ def create(
 class TransportAdapter(HTTPAdapter):
     """Transport adapter that allows customising ssl things"""
 
+    def __init__(self, *args, enable_keep_alive: bool = False, **kwargs):
+        self._enable_keep_alive = enable_keep_alive
+        super().__init__(*args, **kwargs)
+
     def init_poolmanager(
         self, connections, maxsize, block=False, **pool_kwargs
     ):
         self._pool_connections = connections
         self._pool_maxsize = maxsize
         self._pool_block = block
         ssl_context = create_urllib3_context(ciphers=CIPHERS)
+        keep_alive_pool_kwargs = {
+            "socket_options": HTTPConnection.default_socket_options
+            + [
+                (
+                    socket.SOL_SOCKET,
+                    socket.SO_KEEPALIVE,
+                    1,
+                ),  # Enable keep alive.
```

Comment:
Can you have every one of those options as a constant outside, with descriptive names?